ocp/README.md

```diff
@@ -0,0 +1,5 @@
+# COSA as a Custom Build
+
+With the `entry` command, COSA has the wire-framing to become a full-fledged "custom build strategy" for OpenShift.
+
```

Comment:
Note: I'll fill out a complete README.md later. This is provided as the stub for followup.